templates/Gemfile.erb

```diff
@@ -13,7 +14,6 @@ gem "jquery-rails"
 gem "newrelic_rpm", ">= 3.9.8"
 gem "pg"
 gem "rack-canonical-host"
-gem "rails", "<%= Suspenders::RAILS_VERSION %>"
 gem "recipient_interceptor"
 gem "sass-rails", "~> 5.0"
 gem "simple_form"
```

Comment:
I moved rails to the top because I think it's helpful to see the rails version next to the ruby version
When you inject into a file you give a regex as an anchor to inject either `:before` or `:after`